### ISSUE TITLE
alert focus jumps

### ISSUE THREAD

**Issue by jkcox** [ASU-KE/UDS-WordPress-Theme#719]
**Describe the bug**
dismissing alert causes focus to jump, if no element to focus it jumps far around the page.
**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

AB#43962
